src/ApiApprover/PublicApiApprover.cs

```diff
@@ -1,12 +1,15 @@
-﻿using System.IO;
+﻿using System;
+using System.IO;
 using System.Reflection;
 using ApprovalTests;
 using ApprovalTests.Namers;
 
 namespace ApiApprover
 {
+    [Obsolete("The package `ApiApprover` will be removed in the next major version. Install the package `PublicApiGenerator` and use the `ApiGenerator` directly or copy https://github.com/JakeGinnivan/ApiApprover/blob/master/src/ApiApprover/PublicApiApprover.cs into your repository if you plan to continue to use ApprovalTests in combination with the `ApiGenerator`.", false)]
```

Comment:
```suggestion
    [Obsolete("The `ApiApprover` package will be removed in the next major version. Install the `PublicApiGenerator` package and use the `ApiGenerator` class directly instead.", false)]
```

I removed the link to the code because it linked to the master branch, but that code will eventually be removed when ApiApprover is deleted.

If you think that's important to keep, perhaps the example could be added to the README instead, and some sort of sentence saying "To see an example of how to use PublicApiGenerator with ApprovalTests, see the README."